;From: Keith Owens <kaos@ocs.com.au>
;Subject: Patch missing from 2.4.18
;
;2.4.18 is missing a patch to fs/binfmt_elf.c.
;
;diff 2.4.18-rc4 to 2.4.18.
;
Index: 18-rc4.1/fs/binfmt_elf.c
--- 18-rc4.1/fs/binfmt_elf.c Sat, 23 Feb 2002 18:30:39 +1100 kaos (linux-2.4/n/b/2_binfmt_elf 1.1.1.1.1.12.1.3 644)
+++ 18.1/fs/binfmt_elf.c Tue, 26 Feb 2002 07:15:53 +1100 kaos (linux-2.4/n/b/2_binfmt_elf 1.1.1.1.1.12.1.3 644)
@@ -564,9 +564,6 @@ static int load_elf_binary(struct linux_
                        // printk(KERN_WARNING "ELF: Ambiguous type, using ELF\n");
                        interpreter_type = INTERPRETER_ELF;
                }
-       } else {
-               /* Executables without an interpreter also need a personality  */
-               SET_PERSONALITY(elf_ex, ibcs2_interpreter);
        }
 
        /* OK, we are done with that, now set up the arg stuff,


